en/flight_modes_rover/mecanum.md

```diff
@@ -0,0 +1,167 @@
+| ---------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Left stick up/down     | Stick position directly sets speed forward/backward.                                                                                                                                                                   |
+| Left stick left/right  | Stick position sets yaw-rate setpoint. The vehicle will track the setpoint irrespective of uneven surfaces or other external forces. If the input is zero the control system will attempt to maintain a zero yaw rate. |
+| Right stick left/right | Stick position directly sets speed left/right.                                                                                                                                                                         |
+
+For the configuration/tuning of this mode see [Acro mode](../config_rover/mecanum.md#acro-mode).
+
+### Stabilized Mode
+
+::: info
+This mode requires a yaw rate and yaw estimate.
+:::
+
+The vehicle regulates its yaw rate to a setpoint, and a maximum yaw rate can also be specified.
+The vehicle regulates its yaw to a setpoint when the yaw rate setpoint is zero, maintaining the heading.
+Speed is not controlled.
+
+Compared to [Acro mode](#acro-mode), this mode is much better at driving in a straight line as it can more effectively reject disturbances.
+
+| Stick                  | Effect                                                                                                                                                                                                                 |
+| ---------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Left stick up/down     | Stick position directly sets speed forward/backward.                                                                                                                                                                   |
+| Left stick left/right  | Stick position sets yaw-rate setpoint. The vehicle will track the setpoint irrespective of uneven surfaces or other external forces. If this input is zero the control system will maintain the current yaw (heading). |
+| Right stick left/right | Stick position directly sets speed left/right.                                                                                                                                                                         |
+
+For the configuration/tuning of this mode see [Stabilized mode](../config_rover/mecanum.md#stabilized-mode).
```

Comment:
@chfriedrich98 All good fixes, though I did some subedits.

I also changed the way we present some of the manual modes in https://github.com/PX4/PX4-user_guide/pull/3411/commits/e75b1e325a41da4da4c78ed5e099499a16a00a93 which will need checking. This comment uses Stabilized mode as an example of the rest.

Essentially I made that first paragraph summarise everything the mode does. This is then followed by the mode comparision.
Then in the info about the I made this a bit more clear about what stick position sets so that the setpoint ones are more obvious/clear.

I played around with that a bit. Now the table up the top contains the comparative info I did consider making this entirely non-comparative to avoid the repetition. Overall though, I still think this is good.

Reply:
I like the changes you made, IMO this would be ready to be merged.